### DISCUSSION TITLE
not seeing other musicians channel strips in certain rooms

### DISCUSSION THREAD

**Discussion by lrizzo2001** [jamulussoftware/jamulus#3226] [Technical Support & Help] [answered]
So when I go into CBVB room and some others as well, I'm not seeing anybody's channel strips.  I hear everybody and everybody hears me, but I have no faders other than my master fader on the left.  It also does not save my channel strip setting which has my channel strip muted so I don't hear the slap of the return from the app.   I've gone into rooms where this does not happen and I've saved my Mixer Settings and then tried loading them when in the room when this happens and it makes no change.

**Comment by dcorson-ticino-com** [comment]
Jamulus version and OS could be very helpful.
My crystal ball is in for repairs this week.

**Reply by lrizzo2001** [reply]
Mac OS Ventura 13.4.1 (c) 
Jamulus version 3.10.0

**Reply by dcorson-ticino-com** [reply]
I can't really help with a Mac, but this smells like a network problem.
Some UDP packets are not making the journey.
Do you have the possibility to use another provider ?
Maybe using a cellphone hot-spot, just to see if the data connectivity is better (for audio it will be unusable).

**Comment by mcfnord** [comment]
This may be intentional. Do you see any clues in the chat when you (try to) connect?

**Reply by ann0see** [reply]
I think the server sends the whole list back. So it's maybe UDP fragementation related - but then I'd assume that only some sliders not none show up.

**Reply by dcorson-ticino-com** [reply]
It seems to me we have seen this before and made a change so the strangeness of hearing others without seeing their channels doesn't happen. That appears to not be working.  The problem was, if I remember correctly, caused by packets not arriving in the correct order or some not at all.
To find a solution for UDP fragmentation has been on the table for a while, but is over my head.

**Reply by mcfnord** [reply]
I'm 80% sure this is an intentional UX. Ask a server operator where this occurs.

**Reply by dcorson-ticino-com** [reply]
Please give me a hint as to why a server operator would want to not enable the channels, but allow the sound.
I am at a loss.

**Reply by mcfnord** [reply]
The operators I've seen do this when they exclude based on distance (latency too high) or possibly group preference (individuals not allowlisted, or blocklisted). In the past, CBVB ended long-distance connections without notification to the user. People often thought something was wrong with the Jamulus software. A chat message was added that informs the user that they're too far away. Around that time, the "zombie interface" audio also appeared. I've seen this done with sliders and without, but the visitor is unable to add audio, or maybe their audio is mixed up or modified. Ideally a chat message should summarize the situation, so I encourage you to look for one. I believe these tools are used primarily to enforce regional use, so on https://jamulus.live I only show these servers to browsers within a limited geographic distance. Where else do you see this (anti)pattern?

**Reply by dcorson-ticino-com** [reply]
From what I understand from the OPs message the audio is working, but the sliders aren't displayed.  
That is not what you are describing.
I see an error we had in the past where it was possible under certain circumstances to have audio from the server although the client did not have full info from the server to be able to set up the channel.  This was corrected, but we may have a regression here in 3.10... or it is something else...
I agree that it is suspicious that this is happening on the same servers where you saw the behaviour you are describing, but that is not what I read in the OP problem description.

**Comment by lrizzo2001** [comment, marked as answer]
This issue has been confirmed to be tied into the welcome banner when I join the room.  When the amount of text in the welcome banner was reduced, this issue went away.

**Reply by dcorson-ticino-com** [reply]
Well that surely is weird, but may after all be tied UDP fragmentation.
Glad you got a solution.
